ng2-components/ng2-activiti-tasklist/src/services/activiti-tasklist.service.ts

```diff
@@ -73,7 +73,7 @@ export class ActivitiTaskListService {
     getTaskFilterById(taskId: string, appId?: string): Observable<FilterRepresentationModel> {
```

Comment:
Please change the input type instead of cast the response.
Remember to change also the related unit tests.